experimenter/experimenter/experiments/models.py

```diff
@@ -1347,6 +1353,124 @@ def get_branch_data(self, analysis_basis, selected_segment):
 
         return branch_data
 
+    def formattedAnalysisPointComparator(self, point):
+        wi = point.get("window_index")
+        return int(wi) if wi is not None else 0
+
+    def get_metric_area_data(
+        self, metrics, analysis_basis, segment, reference_branch, window="overall"
+    ):
+        metric_data = {}
+        metric_area_data = {"metrics": metrics, "data": metric_data}
+
+        branches = self.sorted_branches
+        overall_results = (
+            (
+                self.results_data.get("v3", {})
+                .get("overall", {})
+                .get(analysis_basis, {})
+                .get(segment, {})
+            )
+            if self.results_data
+            else {}
+        )
+
+        for branch in branches:
+            branch_slug = branch.slug
```

Comment:
Lookups like this cost nothing so storing it locally like this doesn't really help anything